src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,6 @@
+import { createPortal } from 'react-dom';
+import { StyledContainer } from './styles';
+
+export default function ErrorMessage({ error }: { error: string }) {
```

Comment:
Use React.FC type, it has benefits over just declaring return type like you did here.